Cargo.toml

```diff
@@ -31,3 +31,6 @@ inherits = "test"
 package.helix-core.opt-level = 2
 package.helix-tui.opt-level = 2
 package.helix-term.opt-level = 2
+
+[patch.crates-io]
+crossterm = { git = "https://github.com/crossterm-rs/crossterm", rev = "bca71adad70c36de3a4e13d7edc31c0ef9567f47" }
```

Comment:
I could image that any distro packages that include helix (and want to package dependencies separately)  would have trouble with this/reservations.

Does the regression in crossterm `0.26` affect helix even without this PR? If not then distros could just remove this patch and disable the feature.
Otherwise we might consider waiting with merging this PR until the next release cycle (at which point cross-term will hopefully update)? 

I personally think git dependencies like this are fine but it's just something I wanted to draw attention to.

Reply:
Now this patch is used for two things:

* A regression in 0.26 fixed on master that made `terminal::supports_keyboard_enhancement()` always return `Ok(false)`
* Adding support for the `KeyboardEnhancementFlags::REPORT_ALTERNATE_KEYS` flag

It would be fine for a package manager to use 0.26 for the regression, it would just disable this feature permanently. The `KeyboardEnhancementFlags::REPORT_ALTERNATE_KEYS` constant would cause a compile error though since it's new.

If package managers don't like git dependencies then this change should wait for the next crossterm release (or we could publish a new package to crates but that has problems on its own). I wouldn't mind waiting for the next crossterm release

Reply:
I think in that case its fine to merge this as is with a git dependency. Its not a problem for distros to patch out two lines if they take issue with the git dependency :+1: